### PR TITLE
fix doseQuantity  value add location reference (needs better mapping )

### PR DESCRIPTION
**Pull request by jgrant22**
Signed-off-by: jared.grant@ibm.com <jared.grant@ibm.com>

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Turn this to true if you are running/debugging locally so you don't have to build after each change.
-localDevEnv = false
\ No newline at end of file
+localDevEnv = true
```
Comment (LisaWellman) [inline gradle.properties:2]: This should not be delivered

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7ImmunizationFHIRConversionTest.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public void testImmunization() throws IOException {
         String hl7VUXmessageRep = "MSH|^~\\&|MYEHR2.5|RI88140101|KIDSNET_IFL|RIHEALTH|20130531||VXU^V04^VXU_V04|20130531RI881401010105|P|2.5.1|||NE|AL||||||RI543763\r"
                 + "PID|1||12345^^^^MR||TestPatient^Jane^^^^^L||||||\r"
                 + "ORC|RE||197027||ER|||||^Clerk^Myron||MD67895^Pediatric^MARY^^^^MD^^RIA|||||RI2050\r" //ORC.5 is here to prove RXA.20 is taking precedence
-                + "RXA|0|1|20130531|20130531|48^HIB PRP-T^CVX|0.5|ML^^ISO+||00^new immunization record^NIP001|^Sticker^Nurse|^^^RI2050||||33k2a|20131210|PMC^sanofi^MVX|||CP|A|20120901041038\r"
+                + "RXA|0|1|20130531|20130531|48^HIB PRP-T^CVX|0.5|ML^^ISO+||00^new immunization record^NIP001|^Sticker^Nurse|^^^RI2050^ACTIVE||||33k2a|20131210|PMC^sanofi^MVX|||CP|A|20120901041038\r"
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7ImmunizationFHIRConversionTest.java:30]: I do not think we will map 11.5, you can revert this

File: src/test/java/io/github/linuxforhealth/hl7/segments/util/ResourceUtils.java [hunk 3]
```diff
@@ -50,6 +50,7 @@
     public static List<BundleEntryComponent> createFHIRBundleFromHL7MessageReturnEntryList(String inputSegment) {
         HL7ToFHIRConverter ftv = new HL7ToFHIRConverter();
         String json = ftv.convert(inputSegment, OPTIONS);
+        System.out.println(json);
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/util/ResourceUtils.java:53]: do no deliver this

File: src/main/resources/hl7/resource/Immunization.yml [hunk 4]
```diff
@@ -148,4 +149,8 @@ reasonCode:
 recorded:
    type: DATE_TIME
    valueOf: RXA.22
-   expressionType: HL7Spec
\ No newline at end of file
+   expressionType: HL7Spec
+location:
```
Comment (LisaWellman) [inline src/main/resources/hl7/resource/Immunization.yml:153]: Let's remove this and use this branch just for the doseQuantity fix.
For location I think we should map  similar to PV1-3 (where we just did .4 facility)

File: src/main/resources/hl7/resource/Location.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+#
```
Comment (LisaWellman) [inline src/main/resources/hl7/resource/Location.yml:1]: See above comment about facility

**Review by LisaWellman — CHANGES_REQUESTED** [review]

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7ImmunizationFHIRConversionTest.java (hunk 2 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7ImmunizationFHIRConversionTest.java:30]: done

File: src/test/java/io/github/linuxforhealth/hl7/segments/util/ResourceUtils.java (hunk 3 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/util/ResourceUtils.java:53]: thanks

File: src/main/resources/hl7/resource/Immunization.yml (hunk 4 above)
Comment (jgrant22) [reply src/main/resources/hl7/resource/Immunization.yml:153]: removed

File: src/main/resources/hl7/resource/Location.yml (hunk 5 above)
Comment (jgrant22) [reply src/main/resources/hl7/resource/Location.yml:1]: removed

File: src/main/java/io/github/linuxforhealth/hl7/data/SimpleDataValueResolver.java [hunk 6]
```diff
@@ -577,7 +577,15 @@ private static final SimpleCode commonCodingSystemV2 (String table, String code,
         String val = Hl7DataHandlerUtil.getStringValue(value);
         return getFHIRCode(val, AllergyIntoleranceCategory.class);
     };
+    public static final ValueExtractor<Object, String> DOSE_VALUE = (Object value) -> {
```
[the remark this answers is not in the document]
Comment (jgrant22) [reply src/main/java/io/github/linuxforhealth/hl7/data/SimpleDataValueResolver.java:580]: added

File: gradle.properties (hunk 1 above)
Comment (jgrant22) [reply gradle.properties:2]: thanks

**Review by LisaWellman — APPROVED** [review]

**Pull request merged**